.github/workflows/test-ci-cleanup.yml

```diff
@@ -0,0 +1,127 @@
+name: test-ci-cleanup
+on:
+  push:
+    branches:
+      - "jb/QT-400"
+
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '05 02 * * *'
+
+jobs:
+  setup:
+    runs-on: ubuntu-latest
+    outputs:
+      regions: ${{steps.setup.outputs.regions}}
+    steps:
+      - name: Assume service user role via Doormat
+        uses: hashicorp/doormat-action@main
```

Comment:
The workflow fails, claiming this action does not exist, maybe that's since this is the OSS repo. So I'm not sure we can use this pattern, smart though.

Reply:
maybe use the pattern you used in the next job when assuming the creds.

Reply:
this was the end result of trying lots of things to make the if statement work, they all inevitably failed trying to fetch the action (which is private.) So I'll be continuing with your pattern from before.